SignalsIntegrationTests/SignalsIntegrationTests/Infrastructure/DatumComparer.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using Domain;
+
+namespace SignalsIntegrationTests.Infrastructure
+{
+    public class DatumComparer<T> : IComparer
```

Comment:
I'm pretty sure I saw somewhere some 'ad hoc' created comparer. Maybe this one could be also used there?